send_examples.sh

```diff
@@ -3,29 +3,30 @@ i=0
   sleep 1
 done
```

Comment:
Would be probably nice to split a sender script to nice (positive) data and bad (invalid) data. I was initially confused when the agent was reporting errors ;)

Reply:
Fixed by 105d701